design/workitems.go

```diff
@@ -68,6 +68,12 @@ var workItemSingle = JSONSingle(
 	workItem2,
 	workItemLinks)
 
+// workItemReorder is the media type for reorder of work items
+var workItemReorder = JSONReorder(
```

Comment:
Because we need "position" field.

Payload:
'{"data": [{"type": "workitems", "attributes":{"version":0}, "id": "2783"}, {"type": "workitems", "attributes":{"version":0}, "id": "2782"}], "position": {"above":"2770"}}'